chemprop/cli/hpopt.py

```diff
@@ -314,8 +361,16 @@ def tune_model(
         case _:
             raise ValueError(f"Invalid trial scheduler! got: {args.raytune_trial_scheduler}.")
 
+    resources_per_worker = {}
+    if args.raytune_cpu_resources_per_worker:
+        resources_per_worker["CPU"] = args.raytune_cpu_resources_per_worker
```

Comment:
How is the cpu resources per worker different from num cpus? Same question for gpus. Could we instead have an argument for `max_concurrent_trials` and then divide the num cpus by the number of concurrent trials to get cpu resources per worker?

Reply:
num cpus is the total amount of resources ray gets to use (which can have multiple workers), and cpu resources per worker is the amount of resources each worker gets to use. Same for gpus.

I think it's a good idea to use max_concurrent_trials to control this instead. I have modified the code to be:

```

    resources_per_worker = {}
    if args.raytune_num_cpus and args.raytune_max_concurrent_trials:
        resources_per_worker["CPU"] = args.raytune_num_cpus / args.raytune_max_concurrent_trials
    if args.raytune_num_gpus and args.raytune_max_concurrent_trials:
        resources_per_worker["GPU"] = args.raytune_num_gpus / args.raytune_max_concurrent_trials
    if not resources_per_worker:
        resources_per_worker = None
    
    if args.raytune_num_gpus:
        use_gpu = True
    else:
        use_gpu = args.raytune_use_gpu
```